sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/FluxAutoComplete.java

```diff
@@ -72,7 +72,9 @@ static final class AutoCompleteSubscriber extends BaseSubscriber<ServiceBusMessa
 
         @Override
         protected void hookOnSubscribe(Subscription subscription) {
-            logger.info("Subscription received. Subscribing downstream. {}", subscription);
+            logger.atInfo()
```

Comment:
I would downgrade this to verbose. I don't remember using it to diagnose issues often.